src/sql/workbench/services/notebook/common/interface.ts

```diff
@@ -0,0 +1,27 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as azdata from 'azdata';
+import { URI } from 'vs/base/common/uri';
+import { Event } from 'vs/base/common/event';
+
+export interface INotebookInput {
```

Comment:
You're missing a bunch of properties from the original INotebookInput : 
```
	readonly defaultKernel: azdata.nb.IKernelSpec;
	readonly editorOpenedTimestamp: number;
	readonly contentManager: IContentManager;
	readonly standardKernels: IStandardKernelWithProvider[];
```

Reply:
Yeah I realized, that those had code layering issues, but did not see the hygine issues which are being caused without them. I will either move this file or put INotebookInput back into the NotebookService.